internal/resolvers/oembed/facebook.go

```diff
@@ -0,0 +1,72 @@
+package oembed
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+
+	"github.com/Chatterino/api/pkg/utils"
+)
+
+var (
+	facebookAppAccessToken string
+)
+
+func loadFacebookCredentials() (appID string, appSecret string, exists bool) {
+	if appID, exists = utils.LookupEnv("OEMBED_FACEBOOK_APP_ID"); !exists {
+		log.Println("No CHATTERINO_API_OEMBED_FACEBOOK_APP_ID specified, won't do special responses for Facebook or Instagram oEmbed")
+		return
+	}
+
+	if appSecret, exists = utils.LookupEnv("OEMBED_FACEBOOK_APP_SECRET"); !exists {
+		log.Println("No CHATTERINO_API_OEMBED_FACEBOOK_APP_SECRET specified, won't do special responses for Facebook or Instagram oEmbed")
+		return
+	}
```

Comment:
Note for the future, we could try to look around and try extending `utils.LookupEnv` to also print the message for us to avoid duplicating code and avoid potential misspellings in the `log.Println` which could confuse end users.